docker-compose.yml

```
@@ -0,0 +1,96 @@
+    stdin_open: true
+    tty: true
+
+volumes:
+  nats_data:
+    driver: local
+  fpindex_data:
+    driver: local
+
+networks:
+  fpindex-net:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)